conda/hail/meta-template.yaml

```diff
@@ -47,6 +47,7 @@ requirements:
     - google-cloud-sdk
     - google-cloud-storage
     - google-api-core
+    - janus
```

Comment:
Should we pin the version, like: https://github.com/populationgenomics/hail/blob/5659445fd5bca8a15f902f4e41dd8ba19a34757d/hail/python/requirements.txt#L16

Reply:
Good catch, let me pin it

Reply:
Done!